distributor.php

```diff
@@ -17,7 +17,7 @@
 	exit; // Exit if accessed directly.
 }
 
-define( 'DT_VERSION', '1.6.0-dev' );
+define( 'DT_VERSION', '1.6.1' );
```

Comment:
```suggestion
define( 'DT_VERSION', '1.6.1-dev' );
```

Leave the dev in, the build action strips it